PhysicsTools/MVATrainer/plugins/ProcMLP.cc

```diff
@@ -241,7 +241,7 @@ void ProcMLP::trainData(const std::vector<double> *values,
 
 	if (iteration == ITER_COUNT)
 		count++;
-		weightSum += weight;
+   weightSum += weight; 
```

Comment:
No. count++ and weightSum += wight lined up in my editor. Did the author mean to put both statements in curly braces? I went with the way the compiler sees it.